functions.php

```diff
@@ -48,3 +48,20 @@ function setupshop_widgets_init() {
   	'after_title'   => '</h2>', ) );
 }
 ?>
+
+<?php 
+  add_post_meta( 5 , 'setupshop_interventions_focus' , 'education' , false ); 
+  add_post_meta( 5 , 'setupshop_interventions_focus' , 'health' , false );
+  add_post_meta( 5 , 'setupshop_interventions_focus' , 'games' , false );
```

Comment:
The idea is that you can add your meta fields into the back end, not through code otherwise you're going to have to code each page. So your code at this point looks just fine but it's not solving your problem.